test/firebase_auth_mocks_test.dart

```diff
@@ -390,6 +390,25 @@ void main() {
     expect(user.displayName, 'New Bob');
   });
 
+  test('User.updateEmail works', () async {
+    final auth = MockFirebaseAuth(signedIn: true, mockUser: tUser);
+    final user = auth.currentUser;
+    expect(
+      () async => await user!.updateEmail('john@domain.tld'),
```

Comment:
You may be able to rewrite this as:
```dart
expect(
  () => user!.updateEmail(...),
  completion);
```
or
```dart
expectLater(() => user!.updateEmail(...), returnsNormally);
```

Can you try? If it does not work, feel free to leave it as is.

See #70.